camelComponent/README.md

```diff
@@ -0,0 +1,142 @@
+# Overview
+
+This document outlines how to incorporate a Vantiq Camel Component Source into your project. The Camel Component source allows a user to construct Apache Camel applications that interoperate with the Vantiq system. The Camel Component uses the Apache Camel
+mechanism to specify the Vantiq connection details.
+
+The documentation has been split into two parts, [Setting Up Your Machine](#machine)
+and [Setting Up Your Vantiq](#vantiq).
+
+# Prerequisites <a name="pre" id="pre"></a>
+
+**IMPORTANT:** Read the [Testing](#testing) section before building this project.
+
+# Setting Up Your Machine <a name="machine" id="machine"></a>
+
+## Repository Contents
+
+Apache Camel, briefly, is organized into a set of _components_ that are used to produce _endpoints_. 
+In turn, the _endpoints_ can be used in the context of a _consumer_ or _producer_.
+These are from the Apache Camel point of view, so a _producer_ produces messages 
+(Connector publications or queries) for the application, and the _consumer_ takes them in,
```

Comment:
see if it's better (after I commit this).

Reply:
Also, I deleted the phrases about the consumer being at the end since it's not necessarily so. It's logically _at the end_ w/r/t some particular flow, but a route might send stuff to vantiq then log it then send it somewhere else then send something else to vantiq.  Routes with multiple _to_ "operations" are not at all uncommon.